packages/fx-core/resource/package.nls.json

```diff
@@ -328,5 +327,69 @@
   "plugins.apim.error.InvalidPropertyType": "Property '%s' is not type '%s'.",
   "plugins.apim.error.ShortenToEmpty": "The value '%s' cannot be shorten to empty.",
   "plugins.apim.error.UnhandledError": "Unhandled error.",
-  "plugins.apim.error.FailedToParseResourceIdError": "Failed to get '%s' from resource id: '%s'."
+  "plugins.apim.error.FailedToParseResourceIdError": "Failed to get '%s' from resource id: '%s'.",
+  "core.getUserEmailQuestion.validation3": "Email address is not valid",
+  "core.boot.DetailedError": "Detailed error",
```

Comment:
boot?
I think it should be plugins.bot.DetailedError.

Reply:
I guess so. It may under plugins.bot.xor error.bot.* ?

Reply:
fixed, changed to plugins.bot